### PR TITLE
chore: add app triage project github actions

### PR DESCRIPTION
**Pull request by mschile**
- Closes n/a

### User facing changelog

n/a

### Additional details

* Updated the add to triage project GitHub Action to work for forked PRs by changing the trigger to `pull_request_target`
* Added GitHub Action that adds an e2e `routed` item to the e2e project board
  * The action is triggered on `labeled` event for the `routed-to-e2e` label
* Added GitHub Action that moves closed items back to new when a comment is added 
* Added manual GitHub Action that computes the 7-day issue routed/closed metric
  * By default issues from the past 7 days are used, otherwise the user can provide a `startDate` and `endDate`

### Steps to test

n/a

### How has the user experience changed?

n/a

### PR Tasks

- [n/a] Have tests been added/updated?
- [n/a] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: .github/workflows/triage_add_to_routed_project.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Triage add to routed-to project
```
Comment (emilyrohrbough) [inline .github/workflows/triage_add_to_routed_project.yml:1]: I'm not sure what this name is supposed to mean 😬 is it supposed to be something like "Add to routed-to column in Triage Project"?

File: .github/workflows/triage_add_to_routed_project.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-to-e2e:
+    if: github.event.label.name == 'routed-to-e2e'
```
Comment (emilyrohrbough) [inline .github/workflows/triage_add_to_routed_project.yml:8]: any way to update this handle (or is this coming work) dragging issue to column and adding this label & adding to e2e board?

Comment (mschile) [reply .github/workflows/triage_add_to_routed_project.yml:8]: Unfortunately, there aren't any built-in project workflows for changing a fields value and Projects (beta) isn't a trigger event for GitHub Actions.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#project

File: .github/workflows/triage_add_to_routed_project.yml (hunk 1 above)
Comment (emilyrohrbough) [reply .github/workflows/triage_add_to_routed_project.yml:1]: I guess, could it be `Triage: route to team project board`?

Comment (emilyrohrbough) [reply .github/workflows/triage_add_to_routed_project.yml:1]: since this is moving to the correct team-focused project board, not the triage board

**Review by emilyrohrbough — APPROVED** [review]
naming is hard but overall looks great!

**Review by mjhenkes — APPROVED** [review]

File: .github/workflows/triage_add_to_routed_project.yml (hunk 1 above)
Comment (mschile) [reply .github/workflows/triage_add_to_routed_project.yml]: updated naming: [f0f4e68](https://github.com/cypress-io/cypress/pull/23613/commits/f0f4e68f58f82ebed7425418812c89f3abf1eea6)

**Pull request merged**